docs/topics/assembly_developing-and-deploying-spring-reactive-application.adoc

```diff
@@ -0,0 +1,58 @@
+[id='developing-reactive-spring-boot-applications-webflux']
+= Developing reactive {SpringBoot} applications using Spring Reactive and Webflux
```

Comment:
WebFlux is relevant only for a subset of reactive applications. Unless we plan to have a different section which will talk about mail, AMQP, Kafka and whatever else will come in the future, we shouldn't have WebFlux in a main title.
Assuming the project name stays, the title could be "Developing reactive applications with Vert.x Spring Boot"?
What do you think @nainaz

Reply:
Ah, OK, initial understanding was that all of these examples would be based on WebFlux.
Since this not the case, I agree that the title should be _Developing reactive applications with Vert.x Spring Boot_.

Reply:
At the moment all these samples use WebFlux. However, the project itself is not limited to it, e.g. use can use email client on it's own. So to minimise confusion in the future once we expand the documentation, I would prefer to keep it more generic.

Reply:
I've re-titled the section to _Developing reactive applications with Vert.x Spring Boot_.
Since I'me not sure whether _Vert.x Spring Boot_ will be the official name for the Spring+reactive GA release, I'll replace the name with an attribute so we can change it later.

Reply:
UPDATE: According to my conversation with Naina, we should be using _SpringBoot with Vert.X_  as the name.